src/sentry/snuba/discover.py

```diff
@@ -840,6 +840,14 @@ def top_events_timeseries(
             # timestamp needs special handling, creating a big OR instead
             if field == "timestamp":
                 snuba_filter.conditions.append([["timestamp", "=", value] for value in values])
+            # A user field can be any of its field aliases, do an OR across all the user fields
+            elif field == "user":
```

Comment:
Do we need special casing for `issue`?

Reply:
Yup, will make separate PR for that. I think that's the cause of top6 instead of top5 since the conditions aren't strict enough :+1: